storybook/constants.ts

```diff
@@ -11,6 +11,7 @@ export const CATEGORIES = {
 
 export const CATEGORIES_ICON = {
   [CATEGORIES.introduction]: "👋",
+  ["Guides"]: "📚",
```

Comment:
I'm expecting linting to yell at you though haha

```suggestion
  Guides: "📚",
```

Reply:
It didn't work with just the key 😅

Reply:
Haha oh well! Whatever the linter likes 😆

Reply:
For some reason it works now haha